inst/conf/yarn-config-template.yml

```diff
@@ -0,0 +1,9 @@
+default:
```

Comment:
@jjallaire I think if we'll go down with executor.instances to 5, then this is very universal setting and uses very few resources (if we would think about regular yarn cluster with +50 machines).